src/pages/messages/components/PublishingRequestMessagesColumn.tsx

```diff
@@ -35,20 +39,21 @@ export const PublishingRequestMessagesColumn = ({ ticket }: PublishingRequestMes
               <Typography>{new Date(ticket.createdDate).toLocaleDateString()}</Typography>
             )}
           </StyledStatusMessageBox>
-          {ticket.status === 'Pending' || ticket.status === 'New' ? (
-            <StyledStatusMessageBox sx={{ bgcolor: 'secondary.dark' }}>
-              <Typography>{t('registration.files_and_license.files_awaits_approval_unknown')}</Typography>
-            </StyledStatusMessageBox>
-          ) : (
-            <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
-              {ticket.status === 'Completed' ? (
-                <Typography>{t('my_page.messages.files_published')}</Typography>
-              ) : ticket.status === 'Closed' ? (
-                <Typography>{t('my_page.messages.files_rejected')}</Typography>
-              ) : null}
-              {ticket.modifiedDate && <Typography>{new Date(ticket.modifiedDate).toLocaleDateString()}</Typography>}
-            </StyledStatusMessageBox>
-          )}
+          {registrationHasFiles &&
+            (ticket.status === 'Pending' || ticket.status === 'New' ? (
+              <StyledStatusMessageBox sx={{ bgcolor: 'secondary.dark' }}>
+                <Typography>{t('registration.files_and_license.files_awaits_approval_unknown')}</Typography>
+              </StyledStatusMessageBox>
+            ) : (
+              <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
+                {ticket.status === 'Completed' ? (
+                  <Typography>{t('my_page.messages.files_published')}</Typography>
+                ) : (
+                  ticket.status === 'Closed' && <Typography>{t('my_page.messages.files_rejected')}</Typography>
+                )}
+                {ticket.modifiedDate && <Typography>{new Date(ticket.modifiedDate).toLocaleDateString()}</Typography>}
+              </StyledStatusMessageBox>
+            ))}
```

Comment:
Blir ikke enig med meg selv for om det er best å basere dette på `ticket.type` eller `approvedFiles`/`filesForApproval` 🤔 
I teorien åpner vi her for å vise "Fil(er) venter på publisering" selv om alle filer er i `approvedFiles`, så det kan jo bli noen spennende bugs om backend feiler i å holde alt i sync en dag. Men for nå tror jeg det er bra nok sånn 👍